src/main/java/lv/acodemy/homeworks/SecondHomeWork.java

```diff
@@ -0,0 +1,59 @@
+package lv.acodemy.homeworks;
+
+import java.util.Scanner;
+import java.util.Arrays;
+
+public class SecondHomeWork {
+    public static void main(String[] args) {
+        String [] country = {"Latvia", "Estonia", "Lithuania", "Georgia", "Russia",  "Germany", "Turkey", "Poland", "Norway", "Spain"};
+
+
+        for (int i = 0; i < country.length; i++) {
+            System.out.println(country[i]);
+        } if(!country.equals("Lavtia")) {
+            System.out.println("Latvia is my motherland");
+        }
+
+        String [] sentence  = {"Sometimes", "it", "pays", "to", "stay", "in", "bed", "on", "Monday,", "rather", "than", "spending", "the", "rest", "of", "the", "week", "debugging", "Monday’s", "code."};
+
+        System.out.println(Arrays.toString(sentence));
+
+
+
+            /*
+            if(5000=>)
+            if else (5000 10000)
+            else
+             */
+
+        int salary  = 5000;
+        if (salary <= 5000) {
+            System.out.println("Your salary is low.");
+        } else if (salary <= 10000) {
+            System.out.println("Your salary is average.");
+        } else {
+            System.out.println("Your salary is high.");
+        }
```

Comment:
I would like to suggest use else if and add there multiple conditions  so salary value can be checked between 5000 & 1000;